scripts/prometheus_exporter/metrics.js

```diff
@@ -0,0 +1,64 @@
+import { Gauge, Registry } from "https://deno.land/x/ts_prometheus@v0.3.0/mod.ts"
+import { urlJoin } from 'https://deno.land/x/url_join@1.0.0/mod.ts'
+
+const PRUNTIME_URL_BASE = Deno.env.get('PRUNTIME_URL_BASE') ?? 'http://127.0.0.1:8000'
+const PRUNTIME_GET_INFO_URL = urlJoin(PRUNTIME_URL_BASE, '/get_info')
+const PRUNTIME_GET_INFO_POLLING_INTERVAL = parseInt(Deno.env.get('PRUNTIME_GET_INFO_POLLING_INTERVAL') ?? '300')
+const PRUNTIME_GET_INFO_POLLING_TIMEOUT = parseInt(Deno.env.get('PRUNTIME_GET_INFO_POLLING_TIMEOUT') ?? '200')
+
+const sleep = (t) => new Promise(r => setTimeout(r, t))
+
+export const registry = Registry.default
+
+const wrapGauge = (name, help, getterFn) => {
+  const gauge = Gauge.with({ name, help })
+  gauge.set(0)
+  return {
+    gauge,
+    applyInfo: (info) => gauge.set(getterFn(info))
+  }
+}
+const gauges = [
+  wrapGauge('blocknum', 'blocknum', info => info.blocknum),
+  wrapGauge('headernum', 'headernum', info => info.headernum),
+  wrapGauge('para_headernum', 'para_headernum', info => info.para_headernum),
+  wrapGauge('score', 'score', info => info.score),
+  wrapGauge('pending_messages', 'pending_messages', info => info.pending_messages),
+  wrapGauge('running_side_tasks', 'running_side_tasks', info => info.running_side_tasks),
```

Comment:
`running_side_tasks` was removed in the latest pruntime and never used before. So, let's remove this field.

Reply:
updated